components/tools/OmeroPy/test/integration/tablestest/test_tables.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2012-2019 University of Dundee & Open Microscopy Environment.
```

Comment:
Fix dates and I think use `Glencoe Software, Inc.` as in a bunch of other files.